src/generation/generate.rs

```diff
@@ -2996,6 +3003,40 @@ fn gen_spread_element<'a>(node: &SpreadElement<'a>, context: &mut Context<'a>) -
   items
 }
 
+fn maybe_format_tagged_tpl_with_external_formatter<'a>(node: &TaggedTpl<'a>, external_formatter: &ExternalFormatter) -> Option<PrintItems> {
+  let Expr::Ident(ident) = node.tag else {
+    return None;
+  };
+
+  // TODO(bartlomieju): support `html` and `sql` template tags
+  let media_type = match ident.sym().as_str() {
+    "css" => MediaType::Css,
```

Comment:
FYI: Here is the detection code of prettier https://github.com/prettier/prettier/blob/e13614ffd806ee39bdb1ed8fb1026b538de025af/src/language-js/embed/css.js#L72